src/pages/saksbehandling/søknadsbehandling/institusjonsopphold/Institusjonsopphold.tsx

```diff
@@ -91,117 +84,109 @@ const Institusjonsopphold = (props: VilkårsvurderingBaseProps) => {
+                                        feil={fieldState.error?.message}
+                                        value={field.value ?? ''}
+                                    />
+                                )}
                             />
                         </div>
                         {pipe(
                             lagreBehandlingsinformasjonStatus,
                             RemoteData.fold(
                                 () => null,
-                                () => (
-                                    <NavFrontendSpinner>
-                                        {intl.formatMessage({ id: 'display.lagre.lagrer' })}
-                                    </NavFrontendSpinner>
-                                ),
+                                () => <NavFrontendSpinner>{formatMessage('display.lagre.lagrer')}</NavFrontendSpinner>,
                                 () => (
                                     <AlertStripe type="feil">
-                                        {intl.formatMessage({ id: 'display.lagre.lagringFeilet' })}
+                                        {formatMessage('display.lagre.lagringFeilet')}
                                     </AlertStripe>
                                 ),
                                 () => null
                             )
                         )}
                         <Feiloppsummering
-                            tittel={intl.formatMessage({ id: 'feiloppsummering.title' })}
-                            feil={formikErrorsTilFeiloppsummering(formik.errors)}
-                            hidden={!formikErrorsHarFeil(formik.errors)}
+                            tittel={formatMessage('feiloppsummering.title')}
+                            hidden={!isSubmitted || isValid}
+                            feil={hookFormErrorsTilFeiloppsummering(errors)}
+                            innerRef={feiloppsummeringRef}
                         />
                         <Vurderingknapper
                             onTilbakeClick={() => {
                                 history.push(props.forrigeUrl);
                             }}
                             onLagreOgFortsettSenereClick={() => {
-                                formik.validateForm().then((res) => {
-                                    if (Object.keys(res).length === 0) {
-                                        handleSave(
-                                            formik.values,
-                                            Routes.saksoversiktValgtSak.createURL({ sakId: props.sakId })
-                                        );
+                                form.handleSubmit(
+                                    handleSave(Routes.saksoversiktValgtSak.createURL({ sakId: props.sakId })),
+                                    () => {
+                                        setTimeout(() => {
+                                            feiloppsummeringRef.current?.focus();
```

Comment:
hva er greja med å ha en `setTimeout` på 0ms? ☺️

Reply:
Det er et triks/en hack for å få fokuseringen til å kjøre i neste render-loop. Dette er for å sikre at `Feiloppsummering` finnes i DOM-en før vi fokuserer og også for å unngå at første input-felt i formen blir fokusert av react-hook-form.